crates/store/re_chunk/src/chunk.rs

```diff
@@ -74,12 +74,12 @@ impl ChunkComponents {
     ///
     /// I.e semantically equivalent to `get("MyComponent:*.*")`
     #[inline]
-    pub fn get_by_component_name(
+    pub fn get_by_component_type(
```

Comment:
above comment is outdated
might be worth grepping for `component name` (haven't tried myself yet)